### ISSUE TITLE
Problems with SMHS Chromebook software

### ISSUE THREAD

**Issue by AlphaGameDeveloper** [AlphaGameDeveloper/AlphaGameDeveloper.github.io#4]
# Problems with SMHS Chromebook software

## TL;DR
San Marin's Chromebook software, `LightSpeed Filter Agent`, blocked my website alphagame.dev on March 20<sup>th</sup>, 2024.  I am currently working with them to get the website unblocked on the Chromebooks.  **This will not affect anyone that is not at San Marin High School.**

## Why?
I don't know everything, but after emailing the school district, I learned some information in the email response from the IT department. *(Names are hidden for privacy)*
<blockquote>
Hi Damien,
 
We use Lightspeed as our content filter (required by law). Lightspeed is categorizing your website as inappropriate. In my brief review of your website, here are some items that I noticed that may be causing it to be blocked.
 
<ul>
<li>non-educational games</li>
<li>references to hacking</li>
<li>references to virus's and creating virus's</li>
 </ul>

As I mentioned before, your website is still unblocked for staff, so if you have any assignments that you are doing on the website, your teacher can still review/grade them.

Thank you,
-(name removed)
</blockquote>

In Git commit  5a9fd79, I fixed some of their problems with the website, namely, "references to virus's and creating virus's" (no, that is not my typo) by removing WannaDoge from my front page (index.md)

## What's happening now?
I am currently working with the school (I just went to the principal's office to clarify content on the website), so there is still hope of getting this website back on the school chromebooks.

**I will update this GitHub issue with information as it comes.**

**Comment by AlphaGameDeveloper** [comment]
# It's over
I will no longer be using this website for school.  The school decided that I will have another website (made using google sites 🤮) specifically for school.  It can be found [here](https://sites.google.com/students.nusd.org/damienboisvert/home)
